shm/game.cpp

```diff
@@ -424,3 +438,17 @@ void Game::hireAction() {
     }
 
 }
+
+void Game::changeShipsName() {
+    std::cout << "Give new name of your ship\n";
+    std::string newName;
+    std::cin >> newName;
+    ship_->setName(newName);
+}
+
+void Game::playMiniGame() {
+    diceGame dGame;
+    if (dGame.startDiceGame(constValues::pointsToGetMoneyInDiceGame, constValues::moneyForWinInDiceGame)) {
```

Comment:
Maybe pointsToGetMoneyInDiceGame and moneyForWinInDiceGame should be for each island? :)